ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -124,385 +129,392 @@ Control {
         id: stackLayout
         // communities
         anchors.fill:parent
-        ColumnLayout {
+
+        Item {
```

Comment:
I'm wondering if it would be possible / make sense to have all these `Items` for each tab (communities, accounts, collectibles, web..) wrapped on a new external component (like `ProfileDialogXXPanel`) since this file is becoming too big!